### PR TITLE
OpenStack: prompt FIP description

### PR DESCRIPTION
**Pull request by pierreprinetti**
Show Floating IP description/tags in the interactive installer prompt.

In two separate commits:
* the dependency bump
* the OpenStack code

Implements [OSASINFRA-928](https://issues.redhat.com//browse/OSASINFRA-928)

### CODE REVIEW

**Comment by pierreprinetti** [comment]
Assigning following the bot's suggestion, to validate the dependency bump.
image

/assign @sdodson

File: pkg/asset/installconfig/openstack/openstack.go [hunk 1]
```diff
@@ -80,23 +80,22 @@ func Platform() (*openstack.Platform, error) {
...
 		err = survey.Ask([]*survey.Question{
 			{
 				Prompt: &survey.Select{
-					Message: "APIFloatingIPAddress",
-					Help:    "The Floating IP address used for external access to the OpenShift API.",
-					Options: floatingIPNames,
+					Message:     "APIFloatingIPAddress",
+					Help:        "The Floating IP address used for external access to the OpenShift API.",
+					Options:     floatingIPs.Names(),
+					Description: func(_ string, index int) string { return floatingIPs.Description(index) },
```
Comment (mandre) [inline pkg/asset/installconfig/openstack/openstack.go:94]: Nice, I see it only adds a separator when there's an actual description: 
```
❯ ./bin/openshift-install create cluster
? SSH Public Key /home/martin/.ssh/id_rsa.pub
? Platform openstack
? Cloud psi
? ExternalNetwork provider_net_shared_3
? APIFloatingIPAddress  [Use arrows to move, type to filter, ? for more help]
  10.0.108.157
  10.0.108.192
  10.0.109.193
  10.0.109.72
  10.0.110.229
> 10.0.110.52 - jsafrane-psi-api
```

**Review by mandre — COMMENTED** [review]
I see the select description is a recent addition to the survey lib, very à-propos :)

e2e-openstack failure is unrelated.
/lgtm
/approve

**Pull request merged**